pkg/ratelimit/ratelimit.go

```diff
@@ -0,0 +1,82 @@
+package ratelimit
```

Comment:
please add the license blurb like other source files in this repo

Reply:
ok